drivers/net/netdev_upperhalf.c

```diff
@@ -72,6 +84,10 @@ struct netdev_upperhalf_s
 #else
   struct work_s work;
 #endif
+
+#ifdef HAVE_ATOMIC
+  atomic_int quota[NETPKT_TYPENUM]; /* A copy as atomic from lower-half */
```

Comment:
why not change netdev_lowerhalf_s::quota to atomic_int directly in this case?

Reply:
Then the lower-half will need to change their init expression to `atomic_init`. Or maybe we can make a macro for lower-half to init it?

Reply:
is it fine to assign the atomic variable directly in the initial phase?

Reply:
Yes, it seems fine, try to do like this.